v3/cypress/e2e/graph.spec.ts

```diff
@@ -269,9 +269,7 @@ context("Graph UI", () => {
     cy.get("[data-testid=bar-chart-checkbox]").find("input").should("not.be.checked")
     cy.get("[data-testid=bar-cover]").should("not.exist")
     cy.get(".axis-wrapper.left").find("[data-testid=attribute-label]").should("not.exist")
-    // For some reason, at this point Cypress seems to think the checkbox is covered by its container
-    // (`codap-inspector-palette`) so we resort to using `force: true`
-    cy.get("[data-testid=bar-chart-checkbox]").find("input").click({ force: true})
+    cy.get("[data-testid=bar-chart-checkbox]").click()
```

Comment:
Clicking on the label instead of the input works just as well. I should've thought of that before when I ran into this problem. For the record, I believe the reason I had to use `{force: true}` previously is because these Chakra UI-controlled `input` elements have this style rule applied: `clip: rect(0px, 0px, 0px, 0px)`.